Library/.rubocop.yml

```diff
@@ -355,15 +344,18 @@ Style/HashAsLastArrayItem:
     - "/**/Formula/**/*.rb"
     - "**/Formula/**/*.rb"
 
+Style/InverseMethods:
+  InverseMethods:
+    :blank?: :present?
```

Comment:
This, and the `InvertibleUnlessCondition` entry below it, allow us to remove some redundant contents of the `Blank` and `Present` cops (specificallye, the `NotPresent`, `UnlessPresent`, `NotBlank`, and `UnlessBlank` config option handling.